Homework_1.py

```diff
@@ -0,0 +1,43 @@
+# importing libraries
+
+import pandas as pd
+import plotly.express as px
+from sklearn.preprocessing import StandardScaler
+
+data = pd.read_csv("/home/osboxes/Downloads/iris.csv")
+df1 = pd.DataFrame(data)
+df = pd.DataFrame(data)
+df.pop("class")
```

Comment:
Iris.data file doesn't contain any column names. Hence this statement is more of hard-coding.
Did you manually add column names to the Iris.data file ?

Reply:
Yes. I manually added.